eos_data_distribution/ndn/test.py

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/python
+# -*- Mode:python; coding: utf-8; c-file-style:"gnu"; indent-tabs-mode:nil -*- */
+#
+# Copyright (C) 2016 Endless Computers INC.
+# Author: Niv Sardi <xaiki@endlessm.com>
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+# A copy of the GNU Lesser General Public License is in the file COPYING.
+
+from . import file
+from gi.repository import GLib
+import re
+import argparse
+
+import logging
+logging.basicConfig(level=logging.DEBUG)
+
+def process_args(*extra):
+    parser = argparse.ArgumentParser()
+    parser.add_argument("-n", "--name")
+    parser.add_argument("-o", "--output")
```

Comment:
Really? What is being named? What format should the name follow? What’s an output? An output file? Consider that all users are going through the same ramp-up as me.